pkg/pool/pool.go

```diff
@@ -715,6 +736,73 @@ func (p *pool) getPDDeviation() int {
 	return deviationPrefix * NumIPv4AddrPerPrefix
 }
 
+// calculateSecondaryIPDeviation calculates the deviation required to meet the desired state for secondary IP mode
+// Returns a number of IPv4 addresses by taking into account the MinIPTarget and WarmIPTarget
+func (p *pool) calculateSecondaryIPDeviation() int {
+	numWarmResources := numResourcesFromMap(p.warmResources)
+	numUsedResources := len(p.usedResources)
+	numAssignedResources := numUsedResources + numWarmResources + p.pendingCreate + len(p.coolDownQueue)
+
+	// warm pool is in draining state, set targets to zero
+	if p.warmPoolConfig.DesiredSize == 0 {
+		p.log.V(1).Info("DesiredSize is zero, warmPool is in draining state")
```

Comment:
Not sure if these V(1) logs are purely dev purpose. If yes, we can remove them. If they are critical to denug the flow, we should use `log.Info()`.

Reply:
Resolving this after speaking to Hao offline. The consensus is that regardless of logs being potentially useful for troubleshooting,

if the log is deemed critical use log.info(...)
if the log is only useful for debugging, it should be removed